core/object/script_language_extension.h

```diff
@@ -264,6 +264,16 @@ class ScriptLanguageExtension : public ScriptLanguage {
 		}
 	}
 
+	GDVIRTUAL0RC(Vector<String>, _get_doc_comment_block_delimiters)
+
+	virtual void get_doc_comment_block_delimiters(List<Pair<String, String>> *p_words) const override {
+		Vector<String> ret;
+		GDVIRTUAL_CALL(_get_doc_comment_block_delimiters, ret);
+		for (int i = 0; i < ret.size(); i++) {
+			p_words->push_back(Pair<String, String>(ret[i], ""));
+		}
+	}
```

Comment:
I am totally lost about this part. I'm not sure if I implemented it well.

Feedbacks and explanations are very welcome 🙏